projects/canopy/src/lib/card/card-navigation-title/card-navigation-title.component.ts

```diff
@@ -19,12 +20,19 @@ import type { HeadingLevel } from '../../heading';
     class: 'lg-card-navigation-title',
   },
 })
-export class LgCardNavigationTitleComponent {
+export class LgCardNavigationTitleComponent implements OnInit {
   @Input() headingLevel: HeadingLevel;
   @Input() title = '';
   @Input() link = '';
   @Output() linkClickedEvent = new EventEmitter<void>();
 
+  ngOnInit() {
+    if (!(this.headingLevel && this.title && this.link)) {
+      // eslint-disable-next-line no-console
+      console.log('headingLevel, title and link must be set');
```

Comment:
I would change this to `console.error` - it gives more visibility to the user in the console.

Reply:
done 👍